submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/submission/submit/SubmissionDraftApiTest.kt

```diff
@@ -76,5 +76,13 @@ internal class SubmissionDraftApiTest(tempFolder: TemporaryFolder) : BaseIntegra
             webClient.deleteSubmissionDraft(draftSubmission.key)
             assertThat(webClient.getAllSubmissionDrafts()).isEmpty()
         }
+
+        @Test
+        fun `delete submission draft after submission`() {
+            webClient.submitSingle(pageTab, SubmissionFormat.JSON)
+            val draftSubmission = webClient.getSubmissionDraft("ABC-123")
+            webClient.submitSingle(draftSubmission.content.toString(), SubmissionFormat.JSON)
```

Comment:
I think it would be nice to modify this test to perform the same submission defined in the `pageTab` variable but in TSV format, which would also cover the case where a separated "direct submission" that should delete any existing drafts.